src/ctapipe/instrument/optics.py

```diff
@@ -263,3 +269,161 @@ def __repr__(self):
 
     def __str__(self):
         return self.name
+
+
+class PSFModel:
+    def __init__(self, **kwargs):
+        """
+        Base component to describe image distortion due to the optics of the different cameras.
+        """
+
+    @classmethod
+    def subclass_from_name(cls, name, **kwargs):
+        """
+        Obtain an instance of a subclass via its name
+
+        Parameters
+        ----------
+        name : str
+            Name of the subclass to obtain
+
+        Returns
+        -------
+        Instance
+            Instance of subclass to this class
+        """
+
+        subclasses = {base.__name__: base for base in non_abstract_children(cls)}
+
+        requested_subclass = subclasses[name]
+        return requested_subclass(**kwargs)
+
+    @abstractmethod
+    def pdf(self, *args):
+        pass
+
+    @abstractmethod
+    def update_location(self, *args):
+        pass
+
+    @abstractmethod
+    def update_model_parameters(self, *args):
+        pass
+
+
+class ComaModel(PSFModel):
+    """
+    PSF model, describing pure coma aberrations PSF effect
+    """
+
+    def __init__(
+        self,
+        asymmetry_params=[0.49244797, 9.23573115, 0.15216096],
+        radial_scale_params=[0.01409259, 0.02947208, 0.06000271, -0.02969355],
+        az_scale_params=[0.24271557, 7.5511501, 0.02037972],
+    ):
+        """
+        PSF model, describing purely the effect of coma aberration on the PSF
+
+        Parameters
+        ----------
+        asymmetry_params: list
+            Parameters describing the dependency of the asymmetry of the psf on the distance to the center of the camera
```

Comment:
These parameters need better explanations and possibly units.

I suggest to add the relevant formulas to the docstring, see 

https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#directive-math

Reply:
I'll add units where possible,  but for the most part they won't have them. I will add the documentation when I arrive home.

Reply:
this is an ansatz recursive parametrization (parametrization of the parameters) :)
Done empirically by fitting the best-fit values of the PSF parameters for simulations of stars with different offset. The formulas should indeed be provided as a part of the documentation.